qase-cypress/src/index.ts

```diff
@@ -225,7 +230,7 @@ class CypressQaseReporter extends reporters.Base {
                 }
                 if (!this.publishedResults) {
                     this.log(
-                        chalk`{red Could not send all results for 30 seconds after run. Please contact Qase Team.}`
+                        chalk`{red Could not send all results for ${delay} seconds after run. Please contact Qase Team.}`
```

Comment:
did you see <img width="473" alt="image" src="https://user-images.githubusercontent.com/1165416/163257153-7267013a-a446-4211-9543-8e3074ca60c8.png">